media-libs/partio/partio-1.1.0.recipe

```diff
@@ -0,0 +1,95 @@
+SUMMARY="A library for particle IO and manipulation"
+DESCRIPTION="The goal of Partio is to provide a unified interface akin to \
+unified image libraries that makes it easier to load, save, and manipulate \
+particle files.
+
+Major Features:
+
+* Supports Houdini's GEO and BGEO, Maya's PDB and PDA, RenderMan's PTC
+* Arbitrary back end in-memory/cached formats can be supported from a single \
+interface
+* A Python API for easy scripting of particle manipulation
+* A C++ API for high performance particle manipulation
+* Nearest Neighbor Lookups for density estimation, Voronoi computation, etc."
+HOMEPAGE="https://www.disneyanimation.com/technology/partio.html"
+COPYRIGHT="2010-2012 Disney Enterprises"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/wdas/partio/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256='133f386f076bd6958292646b6ba0e3db6d1e37bde3b8a6d1bc4b7809d693999d'
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+PROVIDES="
+	partio$secondaryArchSuffix = $portVersion compat >= 1
+	cmd:partattr = $portVersion compat >= 1
+	cmd:partconv = $portVersion compat >= 1
+	cmd:partinfo = $portVersion compat >= 1
+	cmd:partview = $portVersion compat >= 1
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libGLU$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	partio${secondaryArchSuffix}_devel = $portVersion compat >= 1
+	devel:libpartio$secondaryArchSuffix = $portVersion compat >= 1
+	"
+REQUIRES_devel="
+	partio$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libGL$secondaryArchSuffix
+	devel:libGLU$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:swig
+	"
+
+defineDebugInfoPackage partio$secondaryArchSuffix \
+	$binDir/partattr \
+	$binDir/partconv \
+	$binDir/partinfo \
+	$binDir/partview
```

Comment:
I see that you don't specify `$secondaryArchSuffix` for the `cmd:`s above. If your port installs directly into `$prefix/bin`, please change these into `$prefix/bin` or you will meet some packaging error with `SECONDARY_ARCHITECTURES`